src/core/public/chrome/ui/header/header_logo.tsx

```diff
@@ -90,23 +93,25 @@ interface Props {
   navLinks$: Observable<ChromeNavLink[]>;
   forceNavigation$: Observable<boolean>;
   navigateToApp: (appId: string) => void;
+  loadingCount$?: ReturnType<HttpStart['getLoadingCount$']>;
 }
 
-export function HeaderLogo({ href, navigateToApp, ...observables }: Props) {
+export function HeaderLogo({ href, navigateToApp, loadingCount$, ...observables }: Props) {
   const forceNavigation = useObservable(observables.forceNavigation$, false);
   const navLinks = useObservable(observables.navLinks$, []);
 
   return (
-    <EuiHeaderLogo
-      data-test-subj="logo"
-      iconType="logoElastic"
+    <a
       onClick={(e) => onClick(e, forceNavigation, navLinks, navigateToApp)}
+      className="euiHeaderLogo"
       href={href}
+      data-test-subj="logo"
       aria-label={i18n.translate('core.ui.chrome.headerGlobalNav.goHomePageIconAriaLabel', {
         defaultMessage: 'Go to home page',
```

Comment:
Unrelated nit but maybe you could sneak this in?

```suggestion
        defaultMessage: 'Elastic home',
```

(We don't need to explain that a link takes you somewhere and we never actually say what's in the link so this resolves both while still providing some context on where the link will take them.)